src/foundations/layout/Grid.types.ts

```diff
@@ -1,4 +1,35 @@
-type TSpacingTokens = 0 | 25 | 50 | 75 | 100 | 150 | 200 | 250 | 300 | 350 | 400 | 450 | 500;
+type TGridComponent = 'div' | 'span' | 'section' | 'aside';
+
+type TGridAlignment = 'initial' | 'flex-start' | 'center' | 'flex-end' | 'stretch';
+
+type TGridJustify =
+    | 'initial'
+    | 'flex-start'
+    | 'center'
+    | 'flex-end'
+    | 'stretch'
+    | 'space-around'
+    | 'space-between'
+    | 'space-evenly';
+
+type TGridFlex = number | 'auto';
+
+type TSpacingTokens =
+    | 0
+    | 25
+    | 50
+    | 75
+    | 100
+    | 125
+    | 150
+    | 175
+    | 200
+    | 250
+    | 300
+    | 350
+    | 400
+    | 450
+    | 500;
```

Comment:
I got this directly from the compass design docs: https://zeroheight.com/29be2c109/p/45b751-layout/t/866e5c

Reply:
Sorry, I was referring to the 100, 200, 300 etc., not the actual spacing values. I know you're going for consistency and agree, we just need to bring this suggestion back to design so that the design docs match.

Reply:
Taken a note to discuss with Design